### PR TITLE
[Doxygen Tag] Add report for the plugin @open sesame 04/22 13:38

### PR DESCRIPTION
**Pull request by zhoonit**
- [Doxygen Tag] Add report for the plugin 

```
This patch enables to generate a report for the doxygen tag plugin

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

```

### CODE REVIEW

**Review by gichan-jang — APPROVED** [review]
This change is good for configuring doxygen errors.
LGTM

File: ci/taos/plugins-good/pr-prebuild-doxygen-tag.sh [hunk 1]
```diff
@@ -29,27 +29,29 @@
 ##
 # @brief [MODULE] ${BOT_NAME}/pr-prebuild-doxygen-tag
 function pr-prebuild-doxygen-tag(){
-    echo "########################################################################################"
-    echo "[MODULE] ${BOT_NAME}/pr-prebuild-doxygen-tag: Check if source code includes required Doxygen tags for Doxygen documentation."
+    report_path=../report/doxygen_tag.txt
+
+    echo "########################################################################################" >> $report_path
+    echo "[MODULE] ${BOT_NAME}/pr-prebuild-doxygen-tag: Check if source code includes required Doxygen tags for Doxygen documentation." >> $report_path
```
Comment (dongju-chae) [inline ci/taos/plugins-good/pr-prebuild-doxygen-tag.sh:35]: Q: Previously, this log has been printed to `pr-prebuild-group.txt`. Does this condition still hold?

Comment (zhoonit) [reply ci/taos/plugins-good/pr-prebuild-doxygen-tag.sh:35]: No this won't print to the prebuild-group anymore. I found that others making their own report and not passing report contents to the `pr-prebuild-group.txt` except for the doxygen-tags, so made this for consistency.

Comment (dongju-chae) [reply ci/taos/plugins-good/pr-prebuild-doxygen-tag.sh:35]: I think doxygen-specific report is fine. But, this message can be regarded as a common report.
I can still see similar messages in http://nnstreamer.mooo.com/TAOS-CI/ci/repo-workers/pr-checker/719-202104221338500.35172891616821-bc99590a28b60afffc1b5b8e1b57f2910d18e123/pr-prebuild-group.txt for other prebuild checkers.

I think you may have to consider the uniformity.

Comment (zhoonit) [reply ci/taos/plugins-good/pr-prebuild-doxygen-tag.sh:35]: Understood. I have change those two lines and the status report back to the group :)

**Review by dongju-chae — APPROVED** [review]
LGTM

**Review by myungjoo — APPROVED** [review]

**Pull request merged**